entrypoint.sh

```diff
@@ -57,16 +62,20 @@ cd "$SVN_DIR"
 svn update --set-depth infinity assets
 svn update --set-depth infinity trunk
 
+echo "➤ Current working directory"
+pwd
+
 echo "➤ Copying files..."
 if [[ -e "$GITHUB_WORKSPACE/.distignore" ]]; then
 	echo "ℹ︎ Using .distignore"
 	# Copy from current branch to /trunk, excluding dotorg assets
 	# The --delete flag will delete anything in destination that no longer exists in source
-	rsync -rc --exclude-from="$GITHUB_WORKSPACE/.distignore" "$GITHUB_WORKSPACE/" trunk/ --delete --delete-excluded
+	rsync -rc --exclude-from="$GITHUB_WORKSPACE/.distignore" "$WORKSPACE_DIR/" trunk/ --delete --delete-excluded
```

Comment:
Currently, we're in the `$SNV_DIR`, if `$WORKSPACE_DIR` is set to a relative path, this this command will fail.

Reply:
This and my other PR have been working in since 2020. Thank you for your input. I am closing this PR.